include/vfn/vfio/container.h

```diff
@@ -94,13 +94,11 @@ int vfio_map_vaddr_ephemeral(struct vfio_container *vfio, void *vaddr, size_t le
  */
 int vfio_unmap_ephemeral_iova(struct vfio_container *vfio, size_t len, uint64_t iova);
 
-#ifndef VFIO_IOMMU_TYPE1_INFO_CAP_IOVA_RANGE
-struct vfio_iova_range {
-	__u64	start;
-	__u64	end;
+struct libvfn_iova_range {
```

Comment:
`iommu_iova_range` is the better name. We should use that if defined (i.e. if `linux/iommufd.h` exists). Otherwise, we define our own.

Reply:
I answered all your feedback, but will repeat it in case you missed it.

The thing with iommu_iova_range is that it is already taken in
linux/iommufd.h; that one also has two elements but their names are
different : start , last.
I can probably come up with something where we treat this struct
differently depending on if we are in vfio or iommufd mode